README.md

```diff
@@ -191,19 +192,19 @@ You can specify which model each player should use when running the game. To see
 
 ```bash
 # List all available models
-uv run python -m src.play --list-models
+mind_agents --list-models
```

Comment:
I prefer to encourage UV in the instructions as I think it simplifies installation and makes it much faster

Reply:
this commit brings back the uv commands, 1b728a4a69fa3f884651f58ace6fb044eb399905, the --active command is just in case the end-user have something like pyenv running.